### ISSUE TITLE
iOS 13 Web: Cannot open pages with untrusted certs

### ISSUE THREAD

**Issue by dpgraham** [appium/appium#13185]
## The problem

Test called `should open pages with untrusted certs if the cert was provided in desired capabilities` is not passing

Is not loading the page https://localhost:9762... so the SSL Certificate capability is not working.

**This is probably a lower priority issue. I don't think we need to hold up a release of Appium to support this. If we don't resolve this in time I'd rather just mark it as a known issue**

## Environment

* Appium version (or git revision) that exhibits the issue:
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium:
* Node.js version (unless using Appium.app|exe):
* Npm or Yarn package manager:
* Mobile platform/version under test:
* Real device or emulator/simulator:
* Appium CLI or Appium.app|exe:

## Details

If necessary, describe the problem you have been experiencing in more detail.

## Link to Appium logs

https://gist.github.com/dpgraham/5e185700cb1926ad22b3a94e07ca92a8

## Code To Reproduce Issue [ Good To Have ]

Set the Xcode version to 11

```
git clone https://github.com/appium/appium-xcuitest-driver
git checkout -b ios-13 && git pull origin ios-13
export PLATFORM_VERSION=13.0
export DEVICE_NAME="iPhone X"
```

Go to `safari-ssl-e2e-specs.js` and set `should open pages with untrusted certs if the cert was provided in desired capabilities` to `.only`

```
npm run e2e-test
```

**Comment by dpgraham** [comment]
`should be able to set a secure cookie` test failure is the same issue

**Comment by imurchie** [comment]
To my mind this is what is happening in #13184 too.

**Comment by dpgraham** [comment]
@imurchie I don't think it is.... https://github.com/appium/appium/issues/13184 is getting blocked on https://www.google.com.... this one is not opening the https://localhost:xxxx webpage when passing in SSL

13184 is a higher priority

**Comment by imurchie** [comment]
But async execute relies on having installed a certificate which allows it to hit an ad hoc https server. It gets to google.com but can't get to our localhost https server.

**Comment by umutuzgur** [comment]
@dpgraham Do you have any clue why I get a SIGTERM when I run that test? Do I need to update a certain dependency to make it work?

**Comment by umutuzgur** [comment]
It seems like we found a solution for async execute. We can give low prio to this issue compared to the other iOS 13 stuff

**Comment by KazuCocoa** [comment]
Can we close this?
https://github.com/appium/appium-xcuitest-driver/pull/1073/files has the test fix ( https://support.apple.com/en-us/HT210176 )
